tests/model_hub_tests/models_hub_common/utils.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import os
+import shutil
 import itertools
 import numpy as np
```

Comment:
please do PyCharm formatting

Reply:
I did vs code formatting, I don't think there is a big issue in alphabetical sorting